### PR TITLE
Update cses-1682.mdx, added an alternative solution using SCCs.

### PR DESCRIPTION
**Pull request by Qw3rTy827**

### CODE REVIEW

**Comment by bqi343** [comment]
I don't think it's necessary to include this. Why not just link to the SCC module?

**Review by bqi343 — CHANGES_REQUESTED** [review]
see above

**Comment by bqi343** [comment]
er I meant to link to the advanced module from the solution. I don't see the point of adding this as a problem in the advanced module.

**Comment by Qw3rTy827** [comment]
> er I meant to link to the advanced module from the solution. I don't see the point of adding this as a problem in the advanced module.

so like adding a line that says something along the lines of "Strongly Connected Components" in the Advanced section can also be used to solve this problem and provide a hyperlink to the SCC module ?

**Comment by bqi343** [comment]
sure

**Review by dongliu0426 — APPROVED** [review]

File: solutions/cses-1682.mdx [hunk 1]
```diff
@@ -2,7 +2,7 @@
 id: cses-1682
 source: CSES
 title: Flight Routes Check
-author: Michael Cao, Nathan Gong
+author: Michael Cao, Nathan Gong, Andrew Cheng
```
Comment (dongliu0426) [inline solutions/cses-1682.mdx:5]:
```suggestion
author: Michael Cao, Nathan Gong
```

File: solutions/cses-1682.mdx [hunk 2]
```diff
@@ -192,3 +192,5 @@ public class FlightRoutesCheck {
 </JavaSection>
 
 </LanguageSection>
+
+The problem can also be solved using strongly connected components (SCC). Check it out in the [SCC](/adv/SCC) module.
```
Comment (dongliu0426) [inline solutions/cses-1682.mdx:196]:
```suggestion
The problem can also be solved using [strongly connected components (SCC)](/adv/SCC).
```

**Pull request merged**